### PR TITLE
[ABM] School term

### PR DESCRIPTION
**Pull request by jcostamagna**
## Summary
ABM of School term with summer school included

## Trello Card
https://trello.com/c/raVUUUbI/28-us6-abm-de-schoolterm

### CODE REVIEW

**Comment by jcostamagna** [comment]
retest this please

File: config/routes.rb [hunk 1]
```diff
@@ -7,6 +7,10 @@
   devise_for :teachers
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   api_version(module: 'v1', path: { value: 'api/v1' }, defaults: { format: :json }) do
+    resources :department_staff_sessions, only: [:create]
```
Comment (lmasello) [inline config/routes.rb]: This is no more required

File: app/models/admin.rb [hunk 2]
```diff
@@ -1,11 +1,17 @@
+
 class Admin < User
+  AUTHENTICATION_TOKEN_EXPIRATION_DAYS = Rails.application.secrets.expiration_date_days
```
[the remark this answers is not in the document]
Comment (jcostamagna) [reply app/models/admin.rb]: Done it.

File: app/models/admin.rb [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
   self.table_name = 'admins'
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
   validate :unique_email
 
+  def authentication_token
```
[the remark this answers is not in the document]
Comment (jcostamagna) [reply app/models/admin.rb]: Done it.

File: config/routes.rb (hunk 1 above)
Comment (jcostamagna) [reply config/routes.rb]: Done it.

**Pull request merged**